1-JavaScript/Week4/homework/ex1-doubleEvenNumbers.test.js

```diff
@@ -11,14 +11,10 @@ Let's rewrite it (or _refactor_ it, as experienced developers would call it):
 ------------------------------------------------------------------------------*/
 // ! Function to be tested
 function doubleEvenNumbers(numbers) {
-  // TODO rewrite the function body using `map` and `filter`.
-  const newNumbers = [];
-  for (let i = 0; i < numbers.length; i++) {
-    if (numbers[i] % 2 === 0) {
-      newNumbers.push(numbers[i] * 2);
-    }
-  }
-  return newNumbers;
+  
+  
+  return numbers.filter(number=> number%2===0).map(number=>number*2);
```

Comment:
I love the efficiency, but we don't need to optimise for byte space anymore. We've got strong computers!! 
We lose more cost because of people having hard to time reading code, so lets optimise for readability. Lets add some spaces! 
```suggestion
  return numbers.filter(number=> number % 2 === 0).map(number => number * 2);
```

Reply:
@InouRidder Thank you for your feedback. I tried to use different ways to do the homework; however, your solution is better and easy to read and understand.

I see that you have already adjusted the code. So, do you want me to do any adjustments further?

If not, could you my homework approve it?

And I would appreciate it if you also review my homework for Week3.

Reply:
@InouRidder done